docs/020-deployment/howtos/110-ama-upgrade.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: default
-nav_order: 10
+nav_order: 12
 parent: How-to guides
 grand_parent: Installation
 permalink: /deployment/infra-how-tos/ama-upgrade
-title: AMA upgrade
+title: Marketplace upgrade
```

Comment:
I'd suggest keeping the title "AMA update" as we have an article named "AMA backup and restore". This way the titles are consistent.

Reply:
Yes, that'd make sense. I was thinking from customers perspective. I would argue that most customers wouldn't actually know what 'AMA' means, whereas Marketplace is a more accepted term these days whether it's mobile/pc?

Perhaps we should rename 'AMA backup and restore' to just 'Setup backup and restore'? Maybe outside input required on this one as the backup and restore can also apply to non-AMA installations.

@bkanda @abaciuk any thoughts on this one?

Reply:
If the Marketplace is more familiar to customers, then I agree that the title should be "Marketplace upgrade".

Regarding backup and restore, the article itself is about AMA. I think we can use the name "Marketplace backup and restore".